RFS/README.md

```diff
@@ -91,6 +91,8 @@ When this process completes, it will display the local network URIs for the sour
 +-------------------------+----------------+-----------------+
 ```
 
+By default, this will create an Elasticsearch 7.10 source cluster, but you can get an Elasticsearch 6.8 source cluster using the commands `DocumentsFromSnapshotMigration:es68ComposeUp` and `DocumentsFromSnapshotMigration:es68ComposeDown` instead.
```

Comment:
Interesting from looking at the build.gradle my first instinct wasn't that an es68ComposeUp task would be created but I guess there's some internal linking from the plugin to create these tasks

Reply:
Yeah, check this out - makes it very easy!

https://github.com/avast/gradle-docker-compose-plugin?tab=readme-ov-file#nested-configurations